mlflow/lightgbm.py

```diff
@@ -292,10 +293,40 @@ def autolog():
         - inferred signature of the inputs and outputs of the model.
 
     Note that the `scikit-learn API`_ is not supported.
+
+    :param log_input_example: whether to log a sample of the training data as an example for future
```

Comment:
Nit: Instead of "whether X", can we say: "If `True`, logs a sample of the training data. If `False`, a sample is not logged.` Just to be explicit. Can we make this same tweak for the signature param here, as well as `sklearn.py` and `xgboost.py`?